src/common/AndroidUtils.ts

```diff
@@ -47,16 +46,6 @@ export class AndroidUtils {
         AndroidUtils.logger.setLevel(level);
     }
 
-    /**
-     * Converts a path to UNIX style path.
-     *
-     * @param dirPath Input path.
-     * @returns UNIX style path.
-     */
-    public static convertToUnixPath(dirPath: string): string {
-        return dirPath.replace(/[\\]+/g, '/');
-    }
-
     /**
```

Comment:
This is not Android specific and should have been in `CommonUtils` instead so moved it there.